index.php

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Assignment 6</title>
+  <meta charset="utf-8">
+</head>
+<body>
+<!-- Tony Mainelli
+     assignment6.html
+     INFO2340.WW
+     Thoendel
+     1/21/20
+-->
+<h1>Hello, world!</h1>
+<div>Allow me to introduce myself! My name is <?php echo "Tony Mainelli"; ?>!</div>
```

Comment:
great php statements! complete with quotation marks and those pesky semicolons! 👍